mono/mini/aot-runtime.c

```diff
@@ -5972,7 +5972,7 @@ gpointer
 mono_aot_get_method_checked (MonoDomain *domain,
 							 MonoMethod *method, MonoError *error)
 {
-	error_init (error);
+	//error_init_internal (error); // FIXME? analogous to mono_get_method_checked?
```

Comment:
Why not just removing it?

Reply:
Still trying to determine which are needed.